test/src/kbasesearchengine/test/search/ElasticIndexingStorageTest.java

```diff
@@ -3,6 +3,7 @@
 import static kbasesearchengine.test.common.TestCommon.set;
 import static org.hamcrest.CoreMatchers.is;
 import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
```

Comment:
don't use assertTrue, use assertThat("message", got, is(true));

Reply:
sure. is assertThat better?

Reply:
Basically assert* are all just special cases of assertThat, so by using assertThat you have consistent assertions.